src/ucp/tag/rndv.c

```diff
@@ -748,6 +748,19 @@ static void ucp_rndv_do_rkey_ptr(ucp_request_t *rndv_req, ucp_request_t *rreq,
     ucp_rndv_req_send_ats(rndv_req, rreq, rndv_rts_hdr->sreq.reqptr, status);
 }
 
+static UCS_F_ALWAYS_INLINE int
+ucp_rndv_test_zcopy_scheme_support(size_t length, size_t min_zcopy,
+                                   size_t max_zcopy)
+{
+    return /* is the current message greater than the minimal GET/PUT Zcopy? */
+           (length >= min_zcopy) &&
+           /* is the current message less than the maximal GET/PUT Zcopy? */
+           ((length <= max_zcopy) ||
+           /* or can be split to the segments that are greater than
+            * the minimal GET/PUT Zcopy? */
+            (min_zcopy <= (max_zcopy / 2)));
```

Comment:
maybe we can check this during initialization of rma_bw_lanes? it does not depend on a particular message

Reply:
done